lib/__tests__/disableRanges.test.js

```diff
@@ -990,8 +990,8 @@ function expectDisableRanges(result, expected) {
 
 		expect(actualRanges).toHaveLength(expectedRanges.length);
 
-		for (let i = 0; i < expectedRanges.length; i++) {
-			expectDisableRange(actualRanges[i], expectedRanges[i]);
+		for (const [i, expectedRange] of expectedRanges.entries()) {
```

Comment:
Just note - simple `for` is faster then `for/of` in many cases

Reply:
But it is very small optimizations and most likely will not affect anything, excluding parts that really need to be optimized

Reply:
Yeah, I'm aware of that, but for of is used already in plenty of places in the codebase.

```
C:\Users\xmr\Desktop>matcha bench-loop.js

        125,000,000 ops/sec > for (post) (67.5x)
        110,000,000 ops/sec > for (pre) (59.5x)
        111,000,000 ops/sec > for (cached length) (59.9x)
        111,000,000 ops/sec > forEach (60.1x)
         76,000,000 ops/sec > for of (41.2x)
          2,190,000 ops/sec > for in (1.19x)
          1,850,000 ops/sec > for in (own) (1x)

  Benches: 7
  Fastest: for (post)
  Elapsed: 38.8s
```

That being said, I don't think this is a real issue and readability plus the fact that you can have off by one errors is better :)